src/vester.py

```diff
@@ -0,0 +1,140 @@
+                    url=self.discord_url,
+                )
+            elif tx_hash != HexBytes(0):
+                send_success_to_discord(
+                    tx_type="Release Vested Badger to Tree",
+                    tx_hash=tx_hash,
+                    chain=self.chain,
+                    url=self.discord_url,
+                )
+        except Exception as e:
+            self.logger.error(f"Error processing release tx: {e}")
+            send_error_to_discord(
+                "Badger",
+                "Vest",
+                error=e,
+                chain=self.chain,
+                keeper_address=self.keeper_address,
+            )
+
+    def _send_vest_tx(self) -> HexBytes:
+        """Sends transaction to ETH node for confirmation.
+
+        Raises:
+            Exception: If we have an issue sending transaction (unable to communicate with
+            node, etc.) we log the error and return a tx_hash of 0x00.
+
+        Returns:
+            HexBytes: Transaction hash for transaction that was sent.
+        """
+        tx_hash = HexBytes(0)
+        try:
+            options = {
+                "nonce": self.web3.eth.get_transaction_count(
+                    self.keeper_address, "pending"
+                ),
+                "from": self.keeper_address,
+                "gas": GAS_LIMITS[self.chain],
+            }
+            if self.chain == Network.Ethereum:
+                options["maxPriorityFeePerGas"] = get_priority_fee(self.web3)
+                options["maxFeePerGas"] = self._get_effective_gas_price()
+            else:
+                options["gasPrice"] = self._get_effective_gas_price()
+                self.logger.info(f"max_priority_fee: {self.web3.eth.max_priority_fee}")
+
+            tx = self.vesting_contract.functions.release(
+                CHAIN_CURRENCY[self.chain]
+            ).buildTransaction(options)
+            signed_tx = self.web3.eth.account.sign_transaction(
+                tx, private_key=self.keeper_key
+            )
+            tx_hash = self.web3.eth.send_raw_transaction(signed_tx.rawTransaction)
+        except ValueError as e:
+            self.logger.error(f"Error in sending vesting release tx: {e}")
+            tx_hash = get_hash_from_failed_tx_error(
+                e, self.logger, keeper_address=self.keeper_address
+            )
+        finally:
+            return tx_hash
+
+    def _get_effective_gas_price(self):
+        if self.chain == Network.Polygon:
+            response = requests.get("https://gasstation-mainnet.matic.network").json()
+            gas_price = self.web3.toWei(int(response.get("fast") * 1.1), "gwei")
+        elif self.chain in [Network.Arbitrum, Network.Fantom]:
+            gas_price = int(1.1 * self.web3.eth.gas_price)
+            # Estimated gas price + buffer
+        elif self.chain == Network.Ethereum:
+            # EIP-1559
+            gas_price = get_effective_gas_price(self.web3)
+        return gas_price
```

Comment:
This will error out in case of unknown chain. Need to give gas_price some initial value